mockup/patterns/relateditems/pattern.js

```diff
@@ -378,6 +381,12 @@ define([
 
       $('.dropdown-toggle', self.$toolbar).dropdown();
 
+      // unbind mouseup event from select2 to override the behavior:
+      $(".pattern-relateditems-dropdown").unbind("mouseup");
```

Comment:
👍 nice

Reply:
Yes, but we still need to customize select2, because of the keyboard handling. This is a bit to much to override. Otherwise we would have to copy over a big function to our code. But i made a fork of select2 and set the default branch there to 3.x, so that we can keep track of our customizations.

Reply:
Much better approach, I like it.